src/sagas/session.js

```diff
@@ -67,7 +67,7 @@ export function* listBuckets(getState, action) {
     // that the credentials are valid :/
     yield put(sessionActions.setAuthenticated());
     // Store this valid server url in the history
-    yield put(historyActions.addHistory(getState().session.serverInfo.url));
+    yield put(historyActions.addHistory(getState().session.auth.server));
```

Comment:
This is what was done before but this can be empty when the session has not received this information just yet.

Now I think about it it should never happen, but that's what I saw in the redux logs... I'll investigate